package.json

```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "coffeelint": "^1.9.7",
+    "standard": "^12.0.1",
```

Comment:
Should be `^10.0.3` so that this will dedupe with all our other Standard dependencies.